src/installer/signing/Directory.Build.targets

```diff
@@ -29,6 +29,7 @@
         ArtifactsPackagesDir=$(ArtifactsPackagesDir);
         TargetOS=$(TargetOS);
         TargetArchitecture=$(TargetArchitecture);
+        BuildArchitecture=$(TargetArchitecture);
```

Comment:
Why are we passing `TargetArchitecture` here instead of `BuildArchitecture`?

Reply:
Both are needed. It's to calculate the cross-component paths (in liveBuilds.targets) so they can get properly signed in the two phased manner. This will go away once we get post-build signing.

Reply:
Ok. Sounds good.